addons/l10n_bf/data/account_tax_report_data.xml

```diff
@@ -415,7 +415,12 @@
                         <field name="engine">aggregation</field>
                         <field name="formula">BF_DEDUCTIBLE.balance + BF_CREDIT_REPORTED.balance - BF_CREDIT_ASKED.balance + BF_OTHER_DEDUCTION.balance + BF_UNPAID_CREDIT.balance + BF_OTHER_DEDUCTION.balance - BF_GROSS_TOTAL.balance</field>
                         <field name="subformula">if_above(EUR(0))</field>
-                        <field name="carryover_target">BF_REPORTED.balance</field>
```

Comment:
The fact it is possible to define the field this way is pretty error-prone ; we probably should act against that. 

Let's actually add a constraint on account.report.expression (in a distinct commit if you prefer ; both are fine to me), checking \_carryover_target and label, so that: 

- carryover_target can only be defined if the label starts with _carryover
- carryover_target always references an expression whose label starts with \_applied_carryover_

It could reveal the same error in other l10n modules ; if so, fix them :)

Reply:
Okey I'll do that :)